pkg/crunch42/crunch_test.go

```diff
@@ -0,0 +1,75 @@
+package crunch
+
+import (
+	"fmt"
+	"os"
+	"testing"
+)
+
+var apiKey = ""
+var c *Client
+var collID = ""
+
+func shouldSkip() bool {
+	val, ok := os.LookupEnv("CRUNCH42")
```

Comment:
Please rename this to something like like `CRUNCH42_API_KEY`.

Not a major change but it wasn't obvious to me that it's the API upon first seeing the name of the variable.

Reply:
done, thanks